cfgov/templates/wagtailadmin/css/richtext.css

```diff
@@ -0,0 +1,3 @@
+.richtext {
+  max-width: 1000px
```

Comment:
Gonna get real nitpicky on you here, sorry :)

This should be [indented four spaces](https://github.com/cfpb/development/blob/master/css.md) and should include the semicolon at the end of the line.